src/main/java/pages/PracticePage.java

```diff
@@ -0,0 +1,160 @@
+package pages;
+
+import map.Locators;
+import org.openqa.selenium.*;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PracticePage {
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    public PracticePage(WebDriver driver) {
+        this.driver = driver;
+        this.wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+    }
+
+    // Click Radio3 button
+    public void clickRadioButton3() {
+        WebElement radioButton3 = wait.until(ExpectedConditions.elementToBeClickable(Locators.radiobutton3));
+        radioButton3.click();
+    }
+
+    public boolean isRadioButton3Selected() {
+        WebElement radioButton3 = driver.findElement(Locators.radiobutton3);
+        return radioButton3.isSelected();
+    }
+
+    // Autocomplete - Select India
+    public void selectIndiaFromAutocomplete() {
+        WebElement autocompleteTextbox = driver.findElement(Locators.autocompletebox);
+        autocompleteTextbox.sendKeys("Ind");
+        wait.until(ExpectedConditions.visibilityOfElementLocated(Locators.indiaoption));
+        WebElement suggestion = driver.findElement(Locators.indiaoption);
+        suggestion.click();
+    }
+
+    public String getAutocompleteTextValue() {
+        WebElement autocompleteTextbox = driver.findElement(Locators.autocompletebox);
+        return autocompleteTextbox.getAttribute("value");
+    }
+
+    // Select Option2 from Dropdown
+    public void selectOption2FromDropdown() {
+        WebElement dropdown = driver.findElement(Locators.dropdown);
+        dropdown.click();
+        WebElement option2 = wait.until(ExpectedConditions.elementToBeClickable(Locators.option2));
+        option2.click();
+    }
+
+    public String getSelectedDropdownValue() {
+        WebElement dropdown = driver.findElement(Locators.dropdown);
+        return dropdown.getAttribute("value");
+    }
+
+    // Select Option1 Checkbox
+    public void selectOption1Checkbox() {
+        WebElement checkbox1 = driver.findElement(Locators.checkbox1);
+        checkbox1.click();
+    }
+
+    public boolean isOption1CheckboxSelected() {
+        WebElement checkbox1 = driver.findElement(Locators.checkbox1);
+        return checkbox1.isSelected();
+    }
+
+    // Open New Tab
+    public NewTabPage openNewTab() {
+        driver.findElement(Locators.opentabbtn).click();
+        return new NewTabPage(driver);
+    }
+
+    //Switch Windows
+    public String switchWindowsAndGetTitle() {
+        List<String> tabs = new ArrayList<>(driver.getWindowHandles());
+        driver.switchTo().window(tabs.get(1));
+        return "QAClick Academy - A Testing Academy to Learn, Earn and Shine";
```

Comment:
create separate file as Constant and class to maintain the variables instead of using the value directly in the code